cmd/manager/main.go

```diff
@@ -79,11 +79,11 @@ func main() {
 		cachePath                 string
 		operatorControllerVersion bool
 		systemNamespace           string
-		caCert                    string
+		caCertDir                 string
 	)
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
-	flag.StringVar(&caCert, "ca-cert", "", "The TLS certificate to use for verifying HTTPS connections to the Catalogd web server.")
+	flag.StringVar(&caCertDir, "ca-cert", "", "The directory of TLS certificate to use for verifying HTTPS connections to the Catalogd and docker-registry web servers.")
```

Comment:
1. Should we rename the `ca-cert` flag itself as well?
2. Do I understand correctly that we are changing this to a path to a dir so we can handle multiple CAs? E.g. `/var/certs` will contain CAs of catalogd and OLM generated one?

Reply:
Yes it's a directory and can handle multiple files., although I had thought I could also support specifying a single filename if desired.

Reply:
Ack on the second point.

Back to first point: do you want to keep the flag name as `ca-cert` or rename to something like `ca-dir` or `ca-cert-dir` or something else?

Reply:
I'm ambivalent... but it should probably contain either `dir` or be plural...

Reply:
My vote `ca-certs-dir`.
- `ca-certs` plural, because the directory can contain multiple certificate files
- `dir` because we're asking for a directory (and only a directory, IMO)